sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/CosmosClient.java

```diff
@@ -121,10 +124,18 @@ TokenResolver getTokenResolver() {
         return tokenResolver;
     }
 
+    /**
+     * Gets the cosmos key credential
+     * @return cosmos key credential
+     */
+    CosmosKeyCredential getCosmosKeyCredential() {
```

Comment:
Rest of the getters in CosmosClient use getX() pattern, so wanted to be consistent. 
I think it would make sense to change the rest of them as well, makes sense ?

Reply:
Completely forgot, that it would be a breaking change in that case. I will just change this particular one.